scripts/prepare-for-release.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+PREVIOUS_VERSION=$(awk -F\" '/"version":/ {print $4}' package.json)
+NEW_VERSION=$1
+
+sed -i.bck "s/$PREVIOUS_VERSION/$NEW_VERSION/" package.json
+sed -i.bck "s/$PREVIOUS_VERSION/$NEW_VERSION/" ios/RNPurchases.m
+sed -i.bck "s/$PREVIOUS_VERSION/$NEW_VERSION/" android/build.gradle
+
+yarn example
+pushd examples/purchaseTester
+npx pod-install
+popd
```

Comment:
should we just have them in fastlane, like we do for the other repos? I think the more consistent we can get with our automations, the better

Reply:
makes sense. I figured that it was overkill for a couple of sed commands, but we should do it for when we start adding more automations

Reply:
well, we did have scheduled for this sprint to fully automate deploys for hybrids

Reply:
so it'll help as a foundation for that

Reply:
agreed